backend/models/feedbackModel.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const feedbackSchema = mongoose.Schema(
+  {
```

Comment:
This schema should include the identifier for the user that this feedback belongs to

Reply:
Fixed: Changes are reflected in recent commits